Documentation/ApiOverview/Backend/ContextualMenu.rst

```diff
@@ -94,11 +89,14 @@ After that, a compiled list of items is returned to the
 Menu rendering in JavaScript
 ----------------------------
 
-Based on the JSON data :file:`ContextMenu.js` is rendering a context menu. If
+..  versionchanged:: 12.0
+    The RequireJS module :js:`TYPO3/CMS/Backend/ContextMenuActions` has been migrated
```

Comment:
What does this mean to me?

Reply:
Well it is like with switching from Hooks to Events. The one thing is to learn to apply the new technique, The other thing is that there are new names for all the API. In these comments I I mention the old names once more just like we do mention the old hook when it is replaced by an event?

Reply:
added a link